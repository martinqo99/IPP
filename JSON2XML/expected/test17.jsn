{

"myObject" : {

        "skills": [

                {

                        "category": "Javascript",

                        "tests": [

                                { "name": "One", "score": 90 },

                                { "name": "Two", "score": 96 }

                        ] 

                },

                {

                        "category": "CouchDB",

                        "tests": [

                                { "name": "One", "score": 79 },

                                { "name": "Two", "score": 84 },

                                {

                                "category": "Java",

                                "test": [

                                        { "names": "Three", "score": 950 },

                                        { "name": "Four", "score": 9786 }

                                ] 

                                }

                                

                        ] 

                },

                {

                "interests": [ "Readisdvdng", "Moundvtain Biking", "Hacking", "Reading", "Mountain Bidfvking", "Hacdvking"]

                },

                {

                        "category": "Node.js",

                        "tests": [

                                { "name": "One", "score": 97 },

                                { "name": "Two", "score": 93 }

                        ] 

                }

        ]

}

}