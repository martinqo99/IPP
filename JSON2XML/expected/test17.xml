<?xml version="1.0" encoding="UTF-8"?>

 

<myObject>

    <skills>

        <array>

            <item index="1">

                <tests>

                    <array>

                        <item index="1">

                            <name value="One" />

                            <score value="90" />

                        </item>

                        <item index="2">

                            <name value="Two" />

                            <score value="96" />

                        </item>

                    </array>

                </tests>

                <category value="Javascript" />

            </item>

            <item index="2">

                <tests>

                    <array>

                        <item index="1">

                            <name value="One" />

                            <score value="79" />

                        </item>

                        <item index="2">

                            <name value="Two" />

                            <score value="84" />

                        </item>

                        <item index="3">

                            <test>

                                <array>

                                    <item index="1">

                                        <names value="Three" />

                                        <score value="950" />

                                    </item>

                                    <item index="2">

                                        <name value="Four" />

                                        <score value="9786" />

                                    </item>

                                </array>

                            </test>

                            <category value="Java" />

                        </item>

                    </array>

                </tests>

                <category value="CouchDB" />

            </item>

            <item index="3">

                <interests>

                    <array>

                        <item index="1" value="Readisdvdng" />

                        <item index="2" value="Moundvtain Biking" />

                        <item index="3" value="Hacking" />

                        <item index="4" value="Reading" />

                        <item index="5" value="Mountain Bidfvking" />

                        <item index="6" value="Hacdvking" />

                    </array>

                </interests>

            </item>

            <item index="4">

                <tests>

                    <array>

                        <item index="1">

                            <name value="One" />

                            <score value="97" />

                        </item>

                        <item index="2">

                            <name value="Two" />

                            <score value="93" />

                        </item>

                    </array>

                </tests>

                <category value="Node.js" />

            </item>

        </array>

    </skills>

</myObject>